module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -537,26 +554,23 @@ namespace module::platform::darwin {
                 /************************************************
                  *                  Kinematics Horizon          *
                  ************************************************/
-                Eigen::Affine3d Rwt(sensors->Htw.inverse());
-                // remove translation components from the transform
-                Rwt.translation() = Eigen::Vector3d::Zero();
-                Eigen::Affine3d Rgt(
-                    Eigen::AngleAxisd(-Rwt.rotation().eulerAngles(0, 1, 2).z(), Eigen::Vector3d::UnitZ()) * Rwt);
-                // sensors->Hgt : Mat size [4x4] (default identity)
-                // createRotationZ : Mat size [3x3]
-                // Rwt : Mat size [3x3]
+                // Extract the inverse of the rotation component of Htw
```

Comment:
Do we even use kinematics horizon anywhere anymore? I thought it was gone

Reply:
I don't know. @ysims

Reply:
I don't even know what that is... so no? 😆